src/ChallengeIntroModal3/index.js

```diff
@@ -0,0 +1,78 @@
+import Button from '../Button';
+import ChallengeMeta7 from '../ChallengeMeta7';
+import classNames from 'classnames';
+import React from 'react';
+import Modal from '../Modal';
+import styles from './styles.css';
+import Typography from '../Typography';
+import Wrapper from '../Wrapper';
+const PropTypes = React.PropTypes;
+
+function ChallengeIntroModal3(props) {
+  return (
+    <Modal
+      className={classNames(styles.wrapper, {[props.className]: props.className})}
+      onHide={props.onCancel}
+      show={props.show}
+      style={styles.challengeIntroModal}
+      >
+      <div
+        className={classNames(styles.challengeIntroImage, {[props.className]: props.className})}
+        style={{backgroundImage: `url(${props.image})`}}
+        >
+      </div>
+      <Wrapper
+        className={classNames(styles.challengeIntroWrapper, {[props.className]: props.className})}
+        size="standard"
+        >
+        <Typography
+          style={classNames(styles.tittel)}
+          type="ingressStrong"
+          >
+          {props.children}
+        </Typography>
+        <Typography
+          className={classNames(styles.text)}
+          isYourteam={props.isYourteam}
+          type="bodyTextNormal"
+          >
+          "Du deltar nå på denne utfordringen sammen med {props.memberCount} andre personer!"
+        </Typography>
+        <ChallengeMeta7
+          className={classNames(styles.challengeIntro7)}
+          icon={props.icon}
+          isLeader={props.isLeader}
+          isYourteam={props.isYourteam}
+          memberCount={props.memberCount}
+          name={props.name}
+          theme={props.theme}
+          />
+        <Button
+          className={classNames(styles.button)}
```

Comment:
need to apply the button style `padding-bottom: 16px;` on a div around the button otherwise it adds padding below the button text